sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -30,6 +51,10 @@
   - see the [Azure Identity documentation](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/identity/azure-identity/README.md) for more information
 - `page_number` attribute has been added to `FormTable`
 
+**Dependency updates**
+
+- Adopted [azure-core](https://pypi.org/project/azure-core/) version 1.6.0 or greater
```

Comment:
Should we add this to the setup.py now, or are you waiting until azure-core actually gets released to add it

Reply:
it is added to setup.py